server/model/status_page.js

```diff
@@ -53,6 +54,13 @@ class StatusPage extends BeanModel {
 
         const head = $("head");
 
+        if (statusPage.googleAnalyticsTagId) {
+            let escapedGoogleAnalyticsScript = jsesc(googleAnalytics.getGoogleAnalyticsScript(statusPage.googleAnalyticsTagId), {
```

Comment:
I think jsesc is wrongly used, as `getGoogleAnalyticsScript()` returns with <script> tag which is not purely a js script. I will fix it